lib/application.js

```diff
@@ -618,6 +618,34 @@ app.listen = function listen() {
   return server.listen.apply(server, arguments);
 };
 
+/**
+ * Creates a router that inherits options from app.settings
+ * and allows them to be overriden by passing in an options
+ * object
+ *
+ * @param {Object} options
```

Comment:
We had a PR about this right now, so we should make sure we land it right this time :) The `options` argument needs to be optional in the JSDoc here since it's optional in the code 👍